### PR TITLE
Refactor ProfileManager to use new did-veres-one driver api.

### PR DESCRIPTION
**Pull request by dmitrizagidulin**
Refactor ProfileManager to use new did-veres-one driver api.

### CODE REVIEW

File: did.js [hunk 1]
```diff
@@ -5,34 +5,22 @@
...
 import {DidStore} from 'bedrock-web-did-store';
 import {LDKeyPair} from 'crypto-ld';
-import didv1 from 'did-veres-one';
+import v1 from 'did-veres-one';
 
-const {VeresOneDidDoc} = didv1;
+const veresDriver = v1.driver({mode: 'test'});
 
-export async function generateDid() {
+export async function generateInvokeKey(keyType) {
```
Comment (dlongley) [inline did.js]: This is not specific to the key purpose, so the method name shouldn't include it. Also, we should change the TODO to a FIXME and change `bedrock-web-kms` to `web-kms-client`.

File: did.js [hunk 2]
```diff
@@ -5,34 +5,22 @@
...
-  const v1 = didv1.veres({mode: 'test', keyStore, didStore, metaStore});
-  return v1.generate({invokeKey});
+export async function generateDidDoc({invokeKey, keyType}) {
+  return veresDriver.generate({keyType, invokeKey});
+}
+
+export async function storeDidDocument({dataHub, didDocument}) {
+  const didStore = new DidStore({hub: dataHub});
+
+  // DID store will not store private keys, use KMS instead
```
Comment (dlongley) [inline did.js]: Should this be a FIXME?

File: did.js (hunk 1 above)
Comment (dmitrizagidulin) [reply did.js]: Thanks, done.

File: did.js (hunk 2 above)
Comment (dmitrizagidulin) [reply did.js]: Added.

**Pull request merged**